args.go

```diff
@@ -45,10 +43,10 @@ func ArgsOpt(n int) ArgSpec {
 
 // The global accumulator when Mrb.GetArgs is called. There is a
 // global lock around this so that the access to it is safe.
-var getArgAccumulator []*C.mrb_value
-var getArgLock sync.Mutex
+var getArgAccumulator []C.mrb_value
```

Comment:
I thought I remember a reason we used a pointer here and not the direct value, but can't recall anymore. I suppose it works if you did this, but any idea what that was?